var/spack/repos/builtin/packages/amduprof/package.py

```diff
@@ -3,28 +3,31 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import os
+
 from spack.package import *
 
 
 class Amduprof(Package):
-    """AMD uProf ("MICRO-prof") is a software profiling analysis tool for
-    x86 applications running on Windows, Linux and FreeBSD operating systems
-    and provides event information unique to the AMD "Zen"-based processors
-    and AMD Instinct(tm) MI Series accelerators.
-    """
+    """AMD uProf ("MICRO-prof") is a software profiling analysis tool for x86
+    applications running on Windows, Linux and FreeBSD operating systems and
+    provides event information unique to the AMD "Zen"-based processors and AMD
+    Instinct(tm) MI Series accelerators. AMD uProf enables the developer to better
+    understand the limiters of application performance and evaluate
+    improvements."""
 
     homepage = "https://developer.amd.com/amd-uprof/"
-    url = "https://download.amd.com/developer/eula/uprof/AMDuProf_Linux_x64_4.2.850.tar.bz2"
+    url = f"file://{os.getcwd()}/AMDuProf_Linux_x64_4.2.850.tar.bz2"
+    manual_download = True
```

Comment:
Is there a direct download URL we can use here?  It would be way more convenient for users if they did not have to download the tarball manually.

Reply:
ok let's keep it manual for now, but we can work on getting explicit license acceptance into spack.